dockerExtension.ts

```diff
@@ -124,6 +125,10 @@ export async function activate(ctx: vscode.ExtensionContext): Promise<void> {
 
     ctx.subscriptions.push(vscode.debug.registerDebugConfigurationProvider('docker', new DockerDebugConfigProvider()));
 
+    if (azureAccount) {

```

Comment:
Remove if
Remove setAccount
Move registerCommand to where the other ACR commands are, and use register***Azure***Command